CITATION.cff

```diff
@@ -34,6 +37,9 @@ authors:
     family-names: Young
     affiliation: University College London
     orcid: 'https://orcid.org/0000-0002-8432-7769'
+identifiers:
+  - type: doi
```

Comment:
Does this need `-`? The rest of the file's indented blocks don't seem to use it...